app/scripts/controllers/overview.js

```diff
@@ -410,6 +397,19 @@ angular.module('openshiftConsole')
       });
     };
 
+    $scope.getHPA = function(object) {
+      if (!horizontalPodAutoscalers) {
+        return null;
+      }
+
+      var kind = _.get(object, 'kind'),
+          name = _.get(object, 'metadata.name');
+      if (!_.has($scope, ['hpaByKind', kind, name])) {
+        _.set($scope, ['hpaByKind', kind, name], []);
+      }
+      return _.get($scope, ['hpaByKind', kind, name]);
```

Comment:
It avoids this error from the deploymentDonut $watch.


> angular.js:11706 Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!
Watchers fired in the last 5 iterations: [[{"msg":"fn: regularInterceptedExpression","newVal":[],"oldVal":[]},{"msg":"hpa","newVal":"<<already seen>>","oldVal":"<<already seen>>"}],[{"msg":"fn: regularInterceptedExpression","newVal":[],"oldVal":"<<already seen>>"},{"msg":"hpa","newVal":"<<already seen>>","oldVal":"<<already seen>>"}],[{"msg":"fn: regularInterceptedExpression","newVal":[],"oldVal":"<<already seen>>"},{"msg":"hpa","newVal":"<<already seen>>","oldVal":"<<already seen>>"}],[{"msg":"fn: regularInterceptedExpression","newVal":[],"oldVal":"<<already seen>>"},{"msg":"hpa","newVal":"<<already seen>>","oldVal":"<<already seen>>"}],[{"msg":"fn: regularInterceptedExpression","newVal":[],"oldVal":"<<already seen>>"},{"msg":"hpa","newVal":"<<already seen>>","oldVal":"<<already seen>>"}]]